components/database/postgresql-95/Makefile

```diff
@@ -11,22 +11,27 @@
 #
 # Copyright 2012-2013, EveryCity Ltd. All rights reserved.
 # Copyright 2019, Michal Nowak
+# Copyright 2021 Gary Mills
 #
 
 BUILD_BITS=32_and_64
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		postgresql
+# This release is now unsupported
 COMPONENT_MJR_VERSION=	9.5
-COMPONENT_MNR_VERSION=	24
+COMPONENT_MNR_VERSION=	25
 COMPONENT_VERSION=	$(COMPONENT_MJR_VERSION).$(COMPONENT_MNR_VERSION)
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.bz2
-COMPONENT_ARCHIVE_HASH=	sha256:065cfd3db9f5aca84e794e73e71a797c984b2e728e760f4f4226a9162a99c22a
+COMPONENT_ARCHIVE_HASH=	sha256:7628c55eb23768a2c799c018988d8f2ab48ee3d80f5e11259938f7a935f0d603
 COMPONENT_ARCHIVE_URL=	http://ftp.postgresql.org/pub/source/v$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
 COMPONENT_PROJECT_URL=  https://www.postgresql.org
 
+# Set python version used by this product
+PYTHON_VERSION=3.7
+
 TEST_TARGET=$(TEST_32_and_64)
 
```

Comment:
You don't need to set TEST_TARGET explicitly here. The default derived from BUILD_BITS should be sufficient here.